locales/pt_BR.json

```diff
@@ -0,0 +1,777 @@
+			"fail": "A semântica padrão do elemento não foi sobrescrita com role=\"none\""
+		},
+		"role-presentation": {
+			"pass": "A semântica padrão do elemento foi sobrescrita com role=\"presentation\"",
+			"fail": "A semântica padrão do elemento não foi sobrescrita com role=\"presentation\""
+		},
+		"caption-faked": {
+			"pass": "A primeira linha de uma tabela não é usada como legenda",
+			"fail": "O primeiro elemento da tabela deve ser um <caption> em vez de uma célula da tabela"
+		},
+		"has-caption": {
+			"pass": "A tabela de layout não usa o elemento <caption>",
+			"fail": "A tabela de layout usa o elemento <caption>"
+		},
+		"has-summary": {
+			"pass": "A tabela de layout não usa o atributo 'summary'",
+			"fail": "A tabela de layout usa o atributo 'summary'"
+		},
+		"has-th": {
+			"pass": "A tabela de layout não usa elementos <th>",
+			"fail": "A tabela de layout usa elementos <th>"
+		},
+		"html5-scope": {
+			"pass": "O atributo 'scope' só é utilizado em elementos de cabeçalho de tabela (<th>)",
+			"fail": "No HTML 5, os atributos 'scope' só devem ser utilizados em elementos de cabeçalho de tabela (<th>)"
+		},
+		"same-caption-summary": {
+			"pass": "O conteúdo do atributo 'summary' e <caption> não estão duplicados",
+			"fail": "O conteúdo do atributo 'summary' e <caption> são idênticos"
+		},
+		"scope-value": {
+			"pass": "O atributo 'scope' é usado corretamente",
+			"fail": "O valor do atributo 'scope' pode ser apenas 'row' ou 'col'"
+		},
+		"td-has-header": {
+			"pass": "Todos as células de dados não vazias possuem cabeçalhos",
+			"fail": "Algumas células de dados não vazias não possuem cabeçalho"
+		},
+		"td-headers-attr": {
+			"pass": "O atributo 'headers' é usado exclusivamente para referenciar outras células da tabela",
+			"fail": "O atributo 'headers' não é usado exclusivamente para referenciar outras células da tabela"
+		},
+		"th-has-data-cells": {
+			"pass": "Todas as células de cabeçalho da tabela se referem a células de dados",
+			"fail": "Nem todas as células de cabeçalho da tabela se referem a células de dados",
+			"incomplete": "Células de dados da tabela estão ausentes ou vazias"
+		},
+		"hidden-content": {
+			"pass": "Todo o conteúdo da página foi analisado.",
+			"fail": "Houve problemas ao analisar o conteúdo desta página.",
+			"incomplete": "Existe conteúdo oculto na página que não foi analisado. Será preciso disparar a exibição deste conteúdo para poder analisá-lo."
+		}
+	},
+	"failureSummaries": {
+		"any": {
+			"failureMessage": "Corrija qualquer um dos itens a seguir:{{~it:value}}\n  {{=value.split('\\n').join('\\n  ')}}{{~}}"
+		},
+		"none": {
+			"failureMessage": "Corrija todos os itens a seguir:{{~it:value}}\n  {{=value.split('\\n').join('\\n  ')}}{{~}}"
+		}
+	},
+	"incompleteFallbackMessage": {
+		"undefined": {
+			"failureMessage": "Corrija todos os itens a seguir:{{~it:value}}\n  {{=value.split('\\n').join('\\n  ')}}{{~}}"
```

Comment:
This is not a translation of
"incompleteFallbackMessage": "axe couldn't tell the reason. Time to break out the element inspector!"

But I might be looking at the wrong place

Reply:
Axe message is not actually that. The right one is this:
`"failureMessage": "Fix all of the following:{{~it:value}}\n  {{=value.split('\\n').join('\\n  ')}}{{~}}"`
So, I think the translation is correct.